src/ConditionsResolver/Resovler/Conditional.cs

```diff
@@ -0,0 +1,125 @@
+﻿using System.Globalization;
+
+namespace Monai.Deploy.WorkflowManager.ConditionsResolver.Resolver
+{
+    public class Conditional
+    {
+        public string LogicalOperator { get; set; } = string.Empty;
+
+        public string LeftParameter { get; set; } = string.Empty;
+
+        public string RightParameter { get; set; } = string.Empty;
+
+        public bool RequiresResolving { get; set; } = false;
+
+        public void SetNextParameter(string value)
+        {
+            if (string.IsNullOrEmpty(LeftParameter))
+            {
+                LeftParameter = value;
+                return;
+            }
+            else if (string.IsNullOrEmpty(RightParameter))
+            {
+                RightParameter = value;
+                return;
+            }
+            throw new ArgumentException("All parameters set");
+        }
+
+
+        public bool Parse(ReadOnlySpan<char> input, int currentIndex = 0)
+        {
+            if (currentIndex >= input.Length)
```

Comment:
null check?

Reply:
ReadOnlySpan<char> cant be null. but I've added empty/whitespace checks.